core/src/test/java/org/mule/util/scan/annotations/ClasspathAnnotationsScannerTestCase.java

```diff
@@ -1,42 +0,0 @@
-/*
- * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
- * The software in this package is published under the terms of the CPAL v1.0
- * license, a copy of which has been included with this distribution in the
- * LICENSE.txt file.
- */
-package org.mule.util.scan.annotations;
-
-import org.mule.tck.junit4.AbstractMuleTestCase;
-import org.mule.tck.size.SmallTest;
-
-import org.junit.Test;
-import org.objectweb.asm.ClassReader;
-
-import static org.junit.Assert.assertEquals;
-
-@SmallTest
-public class ClasspathAnnotationsScannerTestCase extends AbstractMuleTestCase
```

Comment:
in case we decide not to delete the tested class, remember to also recover this test